app/models/api/area.rb

```diff
@@ -43,6 +43,10 @@ def self.all_by_area_code
     end
   end
 
+  def self.derived_datasets
+    all_by_area_code.sort_by { |area_code, _| area_code }.map { |_, area| area }.find_all { |area| area.is_derived? }
```

Comment:
Good to know. Thx :+1: 
I guess I should finally really stop using the [Pickaxe](https://www.amazon.com/Programming-Ruby-1-9-2-0-Programmers/dp/1937785491) as my reference doc because it is too outdated. What a pity!